includes/integrations/software-licensing.php

```diff
@@ -44,4 +53,24 @@ private function has_license_check( $status = '' ) {
 		return false;
 	}
 
+	private function has_all_license_check( $status = '' ) {
+		if ( empty( $status ) ) {
+			return false;
+		}
+
+		$license_keys = edd_software_licensing()->get_license_keys_of_user();
+
+		foreach ( $license_keys as $license ) {
+
+			$license_status = edd_software_licensing()->get_license_status( $license->ID );
+
+			if ( $status !== $license_status ) {
```

Comment:
```suggestion
			if ( $status !== $license->status ) {
```

`$license` is already the full license object with the status available, so we don't need a separate function to retrieve it.

Reply:
Agree! Done.